tests/end2end-tests/data_process/test.sh

```diff
@@ -36,6 +36,16 @@ python3 $GS_HOME/tests/end2end-tests/data_process/test_data.py --graph_format Di
 
 error_and_exit $?
 
+# Test the DistDGL graph format with external memory support.
+echo "********* Test the DistDGL graph format with external memory support ********"
+python3 -m graphstorm.gconstruct.construct_graph --conf_file /tmp/test_data/test_data_transform.conf --num_processes 4 --output_dir /tmp/test_partition2 --graph_name test --ext_mem_workspace /tmp --ext_mem_feat_size 2
```

Comment:
Can you add a check to see whether the output of using ext_mem_workspace and with using ext_mem_workspace are same?

Reply:
what is the question?

Reply:
Compare the dgl graphs you build with and without using ext_mem_workspace and check if there is any mismatch.
In theory, using ext_mem_workspace will not change the constructed dgl graph.

Reply:
it's a bit difficult to compare. graph partition has some randomness.